string[] objects
DetectedObject[] ob_msgs
geometry_msgs/Pose2D robot_pose
